packages/twenty-front/src/modules/information-banner/components/reconnect-account/InformationBannerReconnectAccountInsufficientPermissions.tsx

```diff
@@ -21,7 +21,12 @@ export const InformationBannerReconnectAccountInsufficientPermissions = () => {
     reconnect for updates:`}
```

Comment:
simliar to what you did on InformationBannerReconnectAccountEmailAliases, we can safely remove the ```?``` since we already check

```
 if (!accountToReconnect) {
    return null;
  }
```

It has nothing to do with your PR per say, but let's be consistent  !